### DISCUSSION TITLE
version of Maxwell supports by Pyaedt

### DISCUSSION THREAD

**Discussion by Herve-L1** [ansys/pyaedt#4621] [Q&A]
Does anyone know which version of Maxwell is supported by Pyaedt?
Thanks in advance for your help

**Comment by Samuelopez-ansys** [comment]
@Herve-L1 It supports from 2022.R1:
https://aedt.docs.pyansys.com/version/stable/Getting_started/Installation.html#requirements

But it does not mean that the latest PyAEDT releases support 2022R1, you will need to check, if not, you can install the latest PyAEDT versions compatible with old AEDT releases. 

The latest PyAEDT releases are tested in 2024R1, but for example, PyAEDT v0.6.0 was tested in 2023R2.

This is going to be improved, but for now this is how AEDT - PyAEDT compatibility works.

**Comment by Herve-L1** [comment]
Hi Samuel
Thanks for the feedback.
I have version 2022.2 and version 2024.3.
I saw that the installation of the latest version of pyaedt had been done on both versions.
I Will test.

Best regards

Hervé

De : Samuel Lopez ***@***.***>
Envoyé : vendredi 3 mai 2024 10:31
À : ansys/pyaedt ***@***.***>
Cc : Lavalee, Herve (GE Vernova) ***@***.***>; Mention ***@***.***>
Objet : EXT: Re: [ansys/pyaedt] version of Maxwell supports by Pyaedt (Discussion #4621)

AVERTISSEMENT: cet email provient de l'extérieur de GE. Veuillez valider l'adresse e-mail de l'expéditeur avant de cliquer sur les liens ou les pièces jointes, car ils risquent de ne pas être sûrs.

@Herve-L1<https://github.com/Herve-L1> It supports from 2022.R1:
https://aedt.docs.pyansys.com/version/stable/Getting_started/Installation.html#requirements

But it does not mean that the latest PyAEDT releases support 2022R1, you will need to check, if not, you can install the latest PyAEDT versions compatible with old AEDT releases.

The latest PyAEDT releases are tested in 2024R1, but for example, PyAEDT v0.6.0 was tested in 2023R2.

This is going to be improved, but for now this is how AEDT - PyAEDT compatibility works.
